.changeset/old-pandas-scream.md

```diff
@@ -0,0 +1,6 @@
+---
+"@google-labs/breadboard": minor
```

Comment:
Confusingly, since breadboard is on major digit 0, I think this breadboard bump should actually be patch. I think here are the rules to follow:

- If the major digit is > 0:
  - Bugfix: **patch**
  - New feature: **minor**
  - Breaking change: **major**
- If the major digit is 0:
  - Do you want to release 1.0.0?: **major**
  - Bugfix: **patch**
  - New feature: **patch**
  - Breaking change: **minor**

Reply:
```suggestion
"@google-labs/breadboard": patch
```
seems sensible